cluster/src/main/java/org/apache/iotdb/cluster/client/ClientPoolFactory.java

```diff
@@ -54,6 +54,7 @@ public ClientPoolFactory() {
             : new TBinaryProtocol.Factory();
     poolConfig = new GenericKeyedObjectPoolConfig();
     poolConfig.setMaxTotalPerKey(maxConnectionForEachNode);
+    poolConfig.setMaxIdlePerKey(maxConnectionForEachNode / 2);
```

Comment:
Good.
Can we consider setting this parameter in the configuration file?

Reply:
done.